package.json

```diff
@@ -49,6 +49,7 @@
         ]
     },
     "dependencies": {
+        "@eightfold.ai/octuple": "^2.0.1",
```

Comment:
Is this necessary?

Reply:
This came up while installing packages. I'll remove it If it's not required.

Reply:
+1 remove this